components/certificate-mgt/org.wso2.carbon.certificate.mgt.cert.admin.api/src/main/java/org/wso2/carbon/certificate/mgt/cert/jaxrs/api/impl/CertificateManagementAdminServiceImpl.java

```diff
@@ -122,11 +122,14 @@ public Response removeCertificate(@PathParam("serialNumber") String serialNumber
         try {
             boolean status = certificateService.removeCertificate(serialNumber);
             if (!status) {
-                Response.status(Response.Status.NOT_FOUND).entity("No certificate is found with the given " +
-                        "serial number '" + serialNumber + "'");
+                return Response.status(Response.Status.NOT_FOUND).entity(
```

Comment:
Thanks Hasuniea, if I make it a single line it exceeds the number of characters specified to a line. I tried doing it, then the code looks like below.
 "No certificate is found with the given serial number '" + serialNumber + "'").
   build();"